types.js

```diff
@@ -62,6 +62,9 @@ type InAppBrowserAndroidOptions = {
   showInRecents?: boolean,
 };
 
-export type InAppBrowserOptions = InAppBrowserAndroidOptions | InAppBrowseriOSOptions;
+export type InAppBrowserOptions = {
+  ...InAppBrowserAndroidOptions,
```

Comment:
This prevents a missing property toolbarColor error. I think this types makes sense since it includes options from both types, most people probably just pass all options for both platforms.

Reply:
ohh and what's the difference vs `type InAppBrowserOptions = InAppBrowserAndroidOptions & InAppBrowseriOSOptions;`

Reply:
I'm not a flow expert but InAppBrowserAndroidOptions & InAppBrowseriOSOptions didn't work and the other way did haha

Reply:
yeah, that's weird! I prefer TypeScript haha

Reply:
Haha yes! It would probably be easy to convert :)